src/main/java/org/folio/validate/definition/SelectableField.java

```diff
@@ -0,0 +1,92 @@
+package org.folio.validate.definition;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.Validate;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import org.folio.rest.jaxrs.model.CustomField;
+import org.folio.rest.jaxrs.model.SelectField;
+
+@Component
+public abstract class SelectableField {
+
+  @Value("${custom.fields.definition.name.length}")
+  private int NAME_LENGTH_LIMIT;
+
+  @Value("${custom.fields.definition.single.default.size}")
+  private int SINGLE_DEFAULT_SIZE;
+
+  @Value("#{${custom.fields.definition.option.size}}")
+  private Map<String, Integer> optionsSizeMap;
+
+  void validateOptions(CustomField fieldDefinition) {
```

Comment:
I think it's better to have method 
void validateOptions(CustomField fieldDefinition, int maxSize),
That way information specific to each type will be passed from the corresponding validator, instead of being stored as a map in the base class.